README.md

```diff
@@ -99,138 +99,157 @@ outputs, and adhere to any given constraints.
  - Add install directory `C:\Program Files (x86)\Graphviz2.38\bin` to PATH
  
 3. Run the following as admin to configure GraphViz:
-```
-dot -c
-```
+   ```
+   dot -c
+   ```
 
 4. Clone repo (in directory of you choosing)
-```console
-git clone https://github.com/ScottJDaley/ada.git
-```
+   ```cmd
+   git clone https://github.com/ScottJDaley/ada.git
+   ```
 
 5. Move inside the repo
-```console
-cd ada
-```
+   ```cmd
+   cd ada
+   ```
+
+5. Create virtual environment and activate it
+
+   ```cmd
+   py -m venv venv && venv\Scripts\activate
```

Comment:
@ScottJDaley Can you test this syntax on Windows? It should work with both CMD and powershell.

Reply:
works in cmd:
![image](https://user-images.githubusercontent.com/2761011/203464788-91ae398e-6c80-47bd-8ca5-2cec16ca9d23.png)

Not working in powershell:
![image](https://user-images.githubusercontent.com/2761011/203464688-ab5b832f-bf7b-496c-aa82-6b1c6cf937ef.png)

Reply:
This works in powershell though:
![image](https://user-images.githubusercontent.com/2761011/203465041-fb7c8545-e69e-486b-8ad9-6e3a17f1a9e7.png)

Reply:
How about we split this up into two steps so that it is terminal-agnostic?

1. `py -m venv venv`
2. `venv\Scripts\activate`

Reply:
Done.